setup/install/providers/kubernetes-v2/index.md

```diff
@@ -180,6 +175,13 @@ reasons:
   Kubernetes. To avoid building dense and brittle mappings between Spinnaker's
   logical resources and Kubernetes's infrastructure resources, we chose to
   adopt the Kubernetes resources and operations more natively.
+  
+* The V2 provider does __not__ use the [Docker Registry
+  Provider](https://www.spinnaker.io/setup/install/providers/docker-registry/), and we
+  encourage you to stop using the Docker Registry accounts in Spinnaker.  The
```

Comment:
Don't we still need docker registry accounts to trigger pipelines when new docker images arrive?

Reply:
I think the original author's intent here was to clarify that a Kubernetes account no longer requires associated Docker registries (as V1 did to populate Docker images available in the Create Server Group modal), so the Kubernetes provider itself no longer has a dependency on the Docker registry provider...let me know if that makes sense. Suggestions welcome for improved wording, but this diff is just pushing that existing block of text as-is into the bulleted list.

Reply:
Fair enough.  I'm struggling  as I don't see a place that describes triggering pipelines based on the arrival of docker images.  I don't know my way around this repo very well, and my greps are coming up empty.  I sort of expect to see something in https://github.com/spinnaker/spinnaker.github.io/tree/master/guides/user/pipeline/triggers-with-artifactsrewrite or https://github.com/spinnaker/spinnaker.github.io/tree/master/guides/user/pipeline/triggers.  Maybe it's not the end of the world to have no link for the `trigger pipelines` bit.
```
The V2 provider does __not__ use the [Docker Registry
Provider](https://www.spinnaker.io/setup/install/providers/docker-registry/).
You may still need Docker Registry accounts to trigger pipelines, but
otherwise we encourage you to stop using Docker Registry accounts in Spinnaker.
The V2 provider requires that you manage your private registry [configuration
and
authentication](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/)
yourself.
```

Reply:
Sure, thanks @dbyron0! You're officially a Spinnaker docs writer. 🤓